app/views/components/forms/question_types/_hidden_field.html.erb

```diff
@@ -1 +1,4 @@
-<%= hidden_field_tag question.answer_field.to_sym, question.text %>
+<div class="radios" role="group" aria-labelledby="<%= question.answer_field %>">
```

Comment:
Is the div.radios necessary here?

I think the hidden field should be sufficient.

Reply:
Not necessary.  I just like the way the rendered HTML flows with each field sitting in the same type of container.  But I'm cool to strip them out if desired.

Reply:
Ah, okay to leave the div then.

but, perhaps `class` and `role` could be removed.

Reply:
html display attributes removed